chainer/links/normalization/layer_normalization.py

```diff
@@ -0,0 +1,53 @@
+from chainer import initializers
+from chainer import link
+
+from chainer.functions.array import broadcast
+from chainer.functions.math import sqrt
+from chainer.functions.math import square
+from chainer.functions.math import sum
+from chainer.links.connection import scale
+
+
+class LayerNormalization(link.Chain):
+
+    """Layer normalization layer on outputs of linear functions.
+
+    This is a link of "Layer Normalization". This layer
+    normalizes input units by statistics along the second axis,
+    scales and shifts them with :class:`~chainer.links.Scale`.
+
+    Args:
+        size (int): Size of input units.
```

Comment:
I got it. Great!
I will try it, however, current `links.connection.linear.Scale`, used in this Layer Normalization, have not yet supported the parameter shapeholder functionality. So, I will use directly own scaling vector `gamma` and shifting vector `beta` (and `functions.scale`) in place of `Scale` link, and support the parameter shapeholder functionality for them.